test/map.ts

```diff
@@ -178,11 +178,11 @@ test('map.valuesOfType', t => {
 
 	t.throws(() => {
 		ow(new Map([['unicorn', '🦄']]), ow.map.valuesOfType(ow.number));
-	}, '(Map) Expected argument to be of type `number` but received type `string`');
+	}, '(Map) Expected `item` to be of type `number` but received type `string`');
```

Comment:
This is incorrect since we use backticks to indicate variable names or types.

It should ideally be:

```suggestion
	}, '(Map) Expected item 0 to be of type `number` but received type `string`');
```

Reply:
Agree. I'll take some time to see how to achieve that.